### PR TITLE
#165 Add Dependabot configuration

### PR DESCRIPTION
**Pull request by MarekMacko**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: gradle
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (kmaslowiec) [inline .github/dependabot.yml:6]: Would it be better to do it once a week? Daily sounds too bold.  I have real trust issues with that.

Comment (MarekMacko) [reply .github/dependabot.yml:6]: What kind of issues did you encounter?

**Review by BorysChajdas — APPROVED** [review]

**Review by marcinWisnia — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (kmaslowiec) [reply .github/dependabot.yml:6]: We will discuss it during the guild

Comment (MarekMacko) [reply .github/dependabot.yml:6]: Our agreement was to keep everything close to code/PR. Any information about encountered issues is welcome, so don't hesitate to post it here, even if you're not sure if it's valid. That's how we learn from each other 🍻

**Pull request merged**